src/readmg.f

```diff
@@ -174,5 +190,4 @@ SUBROUTINE READMG(LUNXX,SUBSET,JDATE,IRET)
      . ' BE OPEN FOR INPUT')
 901   CALL BORT('BUFRLIB: READMG - INPUT BUFR FILE IS OPEN FOR OUTPUT'//
      . ', IT MUST BE OPEN FOR INPUT')
-902   CALL BORT('BUFRLIB: READMG - ERROR READING A BUFR MESSAGE')
```

Comment:
Did you remove this line intentionally?

Reply:
Yes, because line 902 is no longer accessed anywhere in the source code.  It's a legacy error message left over from when the innards of readmg were changed several years ago, and I just happened to notice that the corresponding line of code was now unreferenced and unreachable.

Reply:
BTW, thanks for your careful review of all of my changes!